deployments/ansible/README.md

```diff
@@ -24,6 +42,8 @@ Note that this role requires root access.
 ```yaml
 - name: Install Splunk OpenTelemetry Connector
   hosts: all
+  # For Windows "become: yes" will raise error.
```

Comment:
What error are you seeing?  become should be supported though we may need to add a platform-dependent `become_user`?: https://docs.ansible.com/ansible/latest/user_guide/become.html#id15

Reply:
Yes, we can add become_user for windows